controls/stig_rhel8.yml

```diff
@@ -70,6 +70,7 @@ controls:
             before granting local or remote access to the system via a command line user logon.
         rules:
             - banner_etc_issue
+            - login_banner_text=dod_banners
```

Comment:
Hi @ggbecker , thanks for reviewing.
Since we didn't have a default content before and STIG is no longer the default since it is very specific, I updated all references I found to STIG which would previously assume in `dod_banners` as default banner. Otherwise, the test scenarios and probably the remediation would fail. However, if this file is not used or there is no intention to use it, we can remove.

Just to confirm, the CIS controls are using this and many other variables as well. I wonder if we will keep any standard among controls. I mean, is expected we remove variables from other controls, include in controls which are missing them or treat each control differently? I would opine by the same standard for all.

Or do you mean only this specific file should be ignored for now?

Reply:
I see now, you are changing the variables file and the default has changed. I guess you can keep it like this. We are not using the STIG controls file yet but we plan to do it in the future. I'm ok with these changes.